lib/variable/include/scipp/variable/variable.tcc

```diff
@@ -55,6 +56,7 @@ auto make_model(const units::Unit unit, const Dimensions &dimensions,
                 element_array<T> values,
                 std::optional<element_array<T>> variances) {
   if constexpr (std::is_same_v<model_t<T>, ElementArrayModel<T>>) {
+    static_assert(!std::is_same_v<T, Eigen::Vector3d>);
```

Comment:
Why specifically vector? What about other structured dtypes?

Reply:
Oh, this is leftover from debugging...